src/Components/CreateImageWizard/formComponents/Repositories.js

```diff
@@ -281,131 +315,177 @@ const Repositories = (props) => {
+                      return 0;
+                    }
+                  })
+                  .slice(computeStart(), computeEnd())
+                  .map((repoURL, rowIndex) => {
+                    const repo = repositories[repoURL];
+                    return (
+                      <Tr key={repo.url}>
+                        <Td
+                          select={{
+                            isSelected: isRepoSelected(repo.url),
+                            rowIndex: rowIndex,
+                            onSelect: (event, isSelecting) =>
+                              handleSelect(repo.url, rowIndex, isSelecting),
+                          }}
+                        />
+                        <Td dataLabel={'Name'}>
+                          {repo.name}
+                          <br />
+                          <Button
+                            component="a"
+                            target="_blank"
+                            variant="link"
+                            icon={<ExternalLinkAltIcon />}
+                            iconPosition="right"
+                            isInline
+                            href={repo.url}
+                          >
+                            {repo.url}
+                          </Button>
+                        </Td>
+                        <Td dataLabel={'Architecture'}>
+                          {repo.distribution_arch}
+                        </Td>
+                        <Td dataLabel={'Version'}>
+                          {repo.distribution_versions}
+                        </Td>
+                        <Td dataLabel={'Packages'}>{repo.package_count}</Td>
+                      </Tr>
+                    );
+                  })}
+              </Tbody>
+            </TableComposable>
+          </>
+        )}
+      </>
+    ))
+  );
+};
+
+const Error = () => {
+  return (
+    <Alert title="Repositories unavailable" variant="danger" isPlain isInline>
+      Repositories cannot be reached, try again later.
+    </Alert>
+  );
+};
+
+const Loading = () => {
+  return (
+    <EmptyState>
+      <EmptyStateIcon variant="container" component={Spinner} />
+      <Title size="lg" headingLevel="h4">
+        Loading
+      </Title>
+    </EmptyState>
+  );
+};
+
+const Empty = ({ isFetching, refetch }) => {
+  const { isBeta } = useGetEnvironment();
+  return (
+    <EmptyState variant={EmptyStateVariant.large} data-testid="empty-state">
+      <EmptyStateIcon icon={RepositoryIcon} />
+      <Title headingLevel="h4" size="lg">
+        No Custom Repositories
+      </Title>
+      <EmptyStateBody>
+        Custom repositories managed via the Red Hat Insights Repositories app
+        will be available here to select and use to search for additional
+        packages.
+      </EmptyStateBody>
+      <Button
+        variant="primary"
+        component="a"
+        target="_blank"
+        href={isBeta() ? '/beta/settings/content' : '/settings/content'}
```

Comment:
Might need to be changed to `/preview/settings/content`

Reply:
Nice catch!